src/Lithnet.GoogleApps/GroupMemberRequestFactory.cs

```diff
@@ -60,7 +60,7 @@ public GroupMembership GetMembership(string groupKey)
                     try
                     {
                         this.WaitForGate();
-                        members = request.ExecuteWithRetryOnBackoff();
+                        members = request.ExecuteWithRetry(RetryEvents.Backoff | RetryEvents.Timeout, ApiExtensions.RetryCount);
```

Comment:
The ApiExtensions.RetryCount parameter is not needed here, as the appropriate overload of ExecuteWithRetry will default to this parameter

Reply:
My bad, I mistook the consumeTokens for the retrycount. Changing.